src/adera/Shaders/PlumeShader.cpp

```diff
@@ -69,14 +69,14 @@ void PlumeShader::draw_plume(
     ACtxPlumeData &rData = *reinterpret_cast<ACtxPlumeData*>(userData[0]);
 
     // Collect uniform data
-    ACompDrawTransform const &drawTf    = rData.m_pDrawTf->get(ent);
+    Matrix4 const &drawTf               = rData.m_pDrawTf->get(ent);
```

Comment:
I guess that I personally find it difficult to notice the ampersand, if it's attached to the variable name. But it's not a big deal